core/src/main/java/com/datastax/oss/driver/api/core/CqlSession.java

```diff
@@ -47,6 +50,9 @@ default ResultSet execute(@Nonnull Statement<?> statement) {
   /**
    * Executes a CQL statement synchronously (the calling thread blocks until the result becomes
    * available).
+   *
+   * @param query The query to execute; never {@code null}.
+   * @return The {@linkplain ResultSet results} returned by the server; never {@code null}.
```

Comment:
I think we tend to omit `@param` / `@return` definitions if it is clear/obvious what the values mean as a way to reduce documentation noise.